Removed extraneous whitespace and fixed Spelling Error

Timestorm opened the pull request:
- Fixed spelling of "opportunities"
- Removed extraneous white-space before "mentorship"
- Removed Oxford commas (customer request)
- Added "and" before "learn new skills"

Commit:
Fixed Whitespace and Spelling Error

axecopfire reviewed (changes requested):
Can you also please update the grammar in the first sentence to have an "and" before new skills please.

Commit:
Fixed spelling and gramatical errors.

Timestorm commented:
> Can you also please update the grammar in the first sentence to have an "and" before new skills please.

Fixed as requested and added a description to the pull request.
